integrationTests/factory/consensusComponents/consensusComponents_test.go

```diff
@@ -30,7 +30,13 @@ func TestConsensusComponents_Close_ShouldWork(t *testing.T) {
 	require.Nil(t, err)
 	managedNetworkComponents, err := node.CreateManagedNetworkComponents(configs, managedCoreComponents)
 	require.Nil(t, err)
-	managedBootstrapComponents, err := node.CreateManagedBootstrapComponents(configs, managedCoreComponents, managedCryptoComponents, managedNetworkComponents)
+	managedBootstrapComponents, err := node.CreateManagedBootstrapComponents(
+		configs,
+		managedCoreComponents,
+		managedCryptoComponents,
+		managedNetworkComponents,
+		make(chan endProcess.ArgEndProcess),
```

Comment:
I think you can reuse the chan from the managedCoreComponents, it should be the same, right? There should be no extra parameter in this method

Reply:
right, we can reuse that one :+1:

Reply:
done

Reply:
ok!